frontend/src/pages/AccountSend.vue

```diff
@@ -115,25 +115,16 @@
             :disable="isSending"
             placeholder="vitalik.eth"
             lazy-rules
+            :hideBottomSpace="true"
             :rules="isValidId"
             ref="recipientIdBaseInputRef"
           />
-          <div
-            class="flex q-pt-sm text-caption warning-container"
-            v-if="recipientIdWarning"
-            :style="!recipientId || isValidRecipientId ? 'margin-top:-2em' : ''"
-          >
+          <div class="flex row text-caption warning-container q-pb-sm" v-if="recipientIdWarning">
             {{ recipientIdWarning }}
           </div>
 
           <!-- Identifier, advanced mode tooltip -->
-          <div
-            v-if="advancedMode"
-            class="row items-center text-caption q-pt-sm q-pb-lg"
-            :style="
-              !recipientId || isValidRecipientId || (!isValidRecipientId && recipientIdWarning) ? 'margin-top:-2em' : ''
-            "
-          >
+          <div v-if="advancedMode" class="row items-center text-caption q-pt-sm">
```

Comment:
(arbitrary thread)

@garyghayrat Yes, it seems like some ascii characters are marked as confusable. m is considered confusable because it is close to rn. I think it is overly strict and it may make sense to create an issue to relax. @mds1 Any thoughts on this?

Reply:
Hmm my initial reaction is to agree it's overly strict for our use case. Does ENS or ensjs have guidance for apps on how they recommend handling this? Wonder what most popular apps do here

Reply:
ENS and ensjs handle detecting confusables in the normalization process. They have an esip and library we could use, but it is heavier than only detecting confusables.

https://docs.ens.domains/ens-improvement-proposals/ensip-15-normalization-standard#wholes

https://github.com/adraffy/ens-normalize.js/tree/main